### PR TITLE
Update tests to use solr 7.7.2

### PR DESCRIPTION
**Pull request by tpendragon**

### CODE REVIEW

File: spec/orangelight/apostrophe_spec.rb [hunk 1]
```diff
@@ -112,7 +112,7 @@
 
   describe 'in left_anchor search' do
     let(:params) do
-      { 'q' => "{!qf=$left_anchor_qf pf=$left_anchor_pf}can’t*" }
+      { qf: "${left_anchor_qf}", pf: "${left_anchor_pf}", q: "can't*" }
```
Comment (carolyncole) [inline spec/orangelight/apostrophe_spec.rb]: Was there a reason to change from the fancy quote to the standard quote?

File: spec/orangelight/cjk_mapping_spec.rb [hunk 2]
```diff
@@ -112,25 +117,28 @@ def add_single_field_doc char
     end
   end
   describe 'mapping applied to search fields' do
+    def qf_pf_params q, field
```
Comment (carolyncole) [inline spec/orangelight/cjk_mapping_spec.rb]: 🎉 for consitency

File: spec/orangelight/apostrophe_spec.rb (hunk 1 above)
Comment (tpendragon) [reply spec/orangelight/apostrophe_spec.rb]: We just didn't notice! Thanks!

**Comment by tampakis** [comment]
Looks great 🎉

**Pull request merged**